app/views/user/show.html.slim

```diff
@@ -39,7 +39,7 @@
         - row.with_value { nil }
       - email_preferences.with_row do |row|
         - row.with_key { 'Email updates about this training course' }
-        - row.with_value(text: @user.training_emails ? t('.training_emails_true') : t('.training_emails_false'), classes: ['data-hj-suppress'])
```

Comment:
This is an example of asking an object multiple questions and hiding the logic in the view layer. If it were a single question on the subject itself it could be reused; another benefit is unit testing. See if you like it.